lib/Utilities/ModuleLoops.cpp

```diff
@@ -14,9 +15,9 @@ GimmeLoops &ModuleLoops::compute(const Function *fcn) {
     // Evil, but okay because NONE of these passes modify the IR
     Function *non_const_function = const_cast<Function *>(fcn);
 
-    // errs() << "Computing loops for " << fcn->getName() << '\n';
+    auto pmt = this->getResolver()->getPMDataManager().getTopLevelManager();
```

Comment:
Add inlined comment pointing to the LLVM commit and explain why this is necessary.